src/RA_Dlg_AchEditor.cpp

```diff
@@ -118,7 +118,7 @@ void Dlg_AchievementEditor::SetupColumns(HWND hList)
     {
         col.mask = LVCF_TEXT | LVCF_WIDTH | LVCF_SUBITEM | LVCF_FMT;
         col.cx = COLUMN_WIDTH[i];
-        tstring colTitle = COLUMN_TITLE[i];	//	Take non-const copy
+        ra::tstring colTitle = COLUMN_TITLE[i];	//	Take non-const copy
         col.pszText = const_cast<LPTSTR>(colTitle.c_str());
```

Comment:
Didn't really work on it since it's unrelated to this PR but in #61 I observed Spectre vulnerabilities when compiling with /Wall. Just thought I'd point it here. Spectre was that thing that reads kernel memory. Most of the places they popped up was with C arrays being assigned or compared.